conanfile.py

```diff
@@ -0,0 +1,47 @@
+from conans import ConanFile, tools, CMake
+
+
+class mediagraphConan(ConanFile):
+    name = "mediagraph"
+    license = "Custom"
+    url = "https://github.com/jpilet/mediagraph"
+    description = "A multi-threaded media pipelining library"
+    author = "Julien Pilet <jpilet@opticode.ch>"
```

Comment:
it is actually julien.pilet@opticode.ch. Thanks.

Reply:
- [ ] fix Julien's email in conan recipe